### DISCUSSION TITLE
Github Education doesn't accept my ID or Transcript. What do I do?

### DISCUSSION THREAD

**Discussion by Mustafa-Tufan** [community/community#140048] [GitHub Education]
### Hi everyone,

Github Education doesn't accept my ID or Transcript. What do I do?

**Comment by tofr** [comment]
The gh crew have gotten much more strict about verification over the past few years. Sometimes there are glitches in their automated system, but the team seems to still be interested in helping you -- even if it may take some time to respond personally to your request. I would ping them and see if you can get a human to assess your credentials.

...which brings me to first point, now at the end: In what manner were your credentials rejected? (Error messages, screens, etc.. )

**Reply by Mustafa-Tufan** [reply]
This is the mail that I received
 
![image](https://github.com/user-attachments/assets/93d41d61-956b-46ad-a57a-403686cf2038)

It doesn't tell much.
But I just realized I sent my transcript in Turkish so I sent a new version with English now, I am waiting for the results.

Edit: Didn't work again :(
